project/Dependencies.scala

```diff
@@ -164,11 +168,13 @@ object Dependencies {
 
   val actor = l ++= Seq(config, java8Compat.value)
 
+  val actorTyped = l ++= Seq(slf4jApi)
+
   val discovery = l ++= Seq(Test.junit, Test.scalatest.value)
 
   val coordination = l ++= Seq(Test.junit, Test.scalatest.value)
 
-  val testkit = l ++= Seq(Test.junit, Test.scalatest.value) ++ Test.metricsAll
+  val testkit = l ++= Seq(Test.logback, Test.junit, Test.scalatest.value) ++ Test.metricsAll
```

Comment:
```suggestion
  val testkit = l ++= Seq(Compile.logback, Test.junit, Test.scalatest.value) ++ Test.metricsAll
```

Reply:
This is classic testkit. Don't know why it was added here. Not needed. Removed.